packages/amplication-client/src/Components/BuildSelector.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  EnumButtonStyle,
+  SelectMenu,
+  SelectMenuItem,
+  SelectMenuList,
+  SelectMenuModal,
+  Label,
+} from "@amplication/design-system";
+import React from "react";
+import { App, Build } from "../models";
+import { BuildSelectorItem } from "./BuildSelectorItem";
+import "./BuildSelector.scss";
+
+const CLASS_NAME = "build-selector";
+
+type Props = {
+  builds: Build[];
+  selectedBuild: Build | null;
+  app: App;
+  onSelectBuild: (commit: Build) => void;
+};
+
+const BuildSelector = ({
```

Comment:
this component should be addressing all the selectors on our app. so it can be more generic and could render any selector. not for this PR